airflow/utils/db_cleanup.py

```diff
@@ -123,7 +126,15 @@ def _check_for_rows(*, query: Query, print_rows=False):
     return num_entities
 
 
-def _do_delete(*, query, orm_model, skip_archive, session):
+def _to_csv(*, target_table, file_path, session):
+    with open(file_path, "w") as f:
```

Comment:
This will silently overwrite any existing file, probably should skip/fail instead?

Reply:
I don't think there would be a case of overwriting since the files are based on timestamps.

Reply:
Ah, I totally missed the timestamp portion! Thanks!